### PR TITLE
[docs] translate menu,popover,popconfirm

### PR DESCRIPTION
**Pull request by wenshiqi0**
translate menu, popover and popconfirm

### CODE REVIEW

**Comment by benjycui** [comment]
[demos](https://github.com/ant-design/ant-design/tree/master/components/menu/demo) should be translated

**Comment by benjycui** [comment]
@visvadw hope that you can review this, thanks 😄

File: components/menu/demo/sider-current.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 2
-title: 只展开当前父级菜单
+title: 
+  zh−CN:只展开当前父级菜单
+  en-US:only to open the current parant menu
```
Comment (visvadw) [inline components/menu/demo/sider-current.md]: en-US: Expand only the current menu (and its parents)

Comment (afc163) [inline components/menu/demo/sider-current.md]:
```diff
- en-US:only
+ en-US: only
```

File: components/menu/demo/sider-current.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+## zh-CN
+
 点击菜单，收起其他展开的所有菜单，保持菜单聚焦简洁。
 
+## en-US
+
+Click the menu, get all the others closed to make the menu simple.
```
Comment (visvadw) [inline components/menu/demo/sider-current.md]: Click the menu and you will see that all the other menus gets collapsed to keep the entire menu compact.

File: components/menu/demo/sider.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 1
-title: 内嵌菜单
+title: 
+  zh-CN:内嵌菜单
+  en-US:inline menu
```
Comment (visvadw) [inline components/menu/demo/sider.md]: en-US: Vertical menu

File: components/menu/demo/sider.md [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+## zh-CN
+
 垂直菜单，子菜单内嵌在菜单区域。
 
+## en-US
+
+Vertical menu which inline the submenu.
```
Comment (visvadw) [inline components/menu/demo/sider.md]: Vertical menu with submenu children.

File: components/menu/demo/switch-mode.md [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Switch the menu type
 ---
 
+## zh-CN
+
 展示动态切换模式。
 
+## en-US
+
+Show the dynamic switching mode.
```
Comment (visvadw) [inline components/menu/demo/switch-mode.md]: Show the dynamic switching mode (between 'inline' and 'vertical').

File: components/menu/demo/theme.md [hunk 6]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 4
-title: 主题
+title: 
+  zh-CN:主题
+  en-US:Theme
```
Comment (visvadw) [inline components/menu/demo/theme.md]: en-US: Menu Themes

File: components/menu/demo/theme.md [hunk 7]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+## zh-CN
+
 内建了两套主题 `light|dark`，默认 `light`。
 
+## en-US
+
+Build in two themes `light|dark`, default `light`.
```
Comment (visvadw) [inline components/menu/demo/theme.md]: There are two built-in themes: 'light' and 'dark'. The default value is 'light'.

File: components/menu/demo/vertical.md [hunk 8]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Vertical menu
 ---
 
+## zh-CN:
+
 子菜单是弹出的形式。
 
+## en-US:
+
+Submenu takes the form of pop-up.
```
Comment (visvadw) [inline components/menu/demo/vertical.md]: Submenus open as pop-ups.

File: components/menu/index.en-US.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+cols: 1
+type: Navigation
+english: Menu
+---
+
+Menu list of Navigation.
+
+## When to use
+
+Navigation menu is important for a website, which makes users jumping from one page to another. Mostly it includes top navigation and side navigation. Top navigation provides all the category and functions of the website. Side navigation provides the Multi-level structure of the website.
```
Comment (visvadw) [inline components/menu/index.en-US.md]: Navigation menu is important for a website, it helps users jump from one site section to another quickly. Mostly, it includes top navigation and side navigation. Top navigation provides all the category and functions of the website. Side navigation provides the Multi-level structure of the website.

File: components/menu/index.en-US.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+## API
+
+### Menu props
+
+| Param    | Description   | Type     | Default value       |
+|----------|---------------|----------|--------------|
+| theme    | color of the theme | String: `light` `dark` | `light` |
+| mode | type of the menu, support vertical, horizontal and inline mode | String: `vertical` `horizontal` `inline` | `vertical` |
```
Comment (visvadw) [inline components/menu/index.en-US.md]: | mode | type of the menu; vertical, horizontal, and inline modes are supported | String: `vertical` `horizontal` `inline` | `vertical` |

File: components/popconfirm/demo/basic.md [hunk 11]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Basic
 ---
 
+## zh-CN
+
 最简单的用法。
 
+## en-US
+
+The basic way.
```
Comment (visvadw) [inline components/popconfirm/demo/basic.md]: The basic example.

File: components/popconfirm/demo/dynamic-trigger.md [hunk 12]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 条件触发
+title: 
+  zh-CN:条件触发
+  en-US:Condition trigger
```
Comment (visvadw) [inline components/popconfirm/demo/dynamic-trigger.md]: en-US: Conditional trigger

File: components/popconfirm/demo/locale.md [hunk 13]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Internationalization
 ---
 
+## zh-CN
+
 设置 `okText` `cancelText` 以自定义按钮文字。
 
+## en-US
+
+Set `okText` `cancelText` to the costom button text.
```
Comment (visvadw) [inline components/popconfirm/demo/locale.md]: Set `okText` and `cancelText` props to customise the buttons' labels.

File: components/popconfirm/demo/placement.md [hunk 14]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Placement
 ---
 
+## zh-CN
+
 位置有十二个方向。
 
+## en-US
+
+The placement has twelve direction.
```
Comment (visvadw) [inline components/popconfirm/demo/placement.md]: There are 12 `placement` options available.

File: components/popconfirm/index.en-US.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
+---
+category: Components
+chinese: 气泡确认框
+type: Views
+english: Popconfirm
+---
+
+The bubble confirmation box popped by clicking.
```
Comment (visvadw) [inline components/popconfirm/index.en-US.md]: A simple and compact dialog used for confirmation purposes.

File: components/popconfirm/index.en-US.md [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+
+## When to use
+
+For the further confirmation, pop a float tip box to ask.
```
Comment (visvadw) [inline components/popconfirm/index.en-US.md]: A simple and compact dialog used for asking an user confirmation.

File: components/popover/demo/basic.md [hunk 17]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Basic
 ---
 
+## zh-CN
+
 最简单的用法，浮层的大小由内容区域决定。
 
+## en-US
+
+The most basic way. The size of the floating layer depends on the region of contents.
```
Comment (visvadw) [inline components/popover/demo/basic.md]: The most basic example. The size of the floating layer depends on the region of contents.

File: components/popover/demo/control.md [hunk 18]
```diff
@@ -1,10 +1,18 @@
 ---
 order: 3
-title: 从浮层内关闭
+title: 
+  zh-CN:从浮层内关闭
+  en-US:close in the floating layer
```
Comment (visvadw) [inline components/popover/demo/control.md]: en-US: Controlling the close of the dialog

File: components/popover/demo/control.md [hunk 19]
```diff
@@ -1,10 +1,18 @@
...
 ---
 
+## zh-CN
+
 使用 `visible` 属性控制浮层显示。
 
+## en-US
+
+use `visible` attribute to control the display of the floating layer.
```
Comment (visvadw) [inline components/popover/demo/control.md]: Use `visible` prop to control the display of the dialog.

File: components/popover/demo/placement.md [hunk 20]
```diff
@@ -1,10 +1,18 @@
...
+  en-US:Placement
 ---
 
+## zh-CN
+
 位置有十二个方向。
 
+## en-US
+
+The placement has twelve direction.
```
Comment (visvadw) [inline components/popover/demo/placement.md]: There are 12 `placement` options available.

File: components/popover/index.en-US.md [hunk 21]
```diff
@@ -0,0 +1,29 @@
+---
+category: Components
+chinese: 气泡卡片
+type: Views
+english: Popover
+---
+
+The bubble float card popped by clicking or hovering.
```
Comment (visvadw) [inline components/popover/index.en-US.md]: The floating card popped by clicking or hovering.

File: components/popover/index.en-US.md [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+chinese: 气泡卡片
+type: Views
+english: Popover
+---
+
+The bubble float card popped by clicking or hovering.   
+
+## When to use
+
+For more information or operations, put all into the popover card to show according to the behavior.
```
Comment (visvadw) [inline components/popover/index.en-US.md]: [todo] What do you mean by that?

File: components/popover/index.en-US.md [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
+
+The difference with `Tooltip` is the popover card can provide operations and much more complicated information such as link and button.
```
Comment (visvadw) [inline components/popover/index.en-US.md]: Comparing with `Tooltip`, besides information `Popover` card can also provide action elements like links and buttons.

File: components/popover/index.en-US.md [hunk 24]
```diff
@@ -0,0 +1,29 @@
...
+
+## API
+
+| Param     | Description   | Type     | Default value       |
+|-----------|------------------------------------------|---------------|--------|
+| trigger   | trigger for behavior, optional `hover/focus/click`       | string        | hover  |
```
Comment (visvadw) [inline components/popover/index.en-US.md]: | trigger   | triggering mode: can be `hover`, `focus`, or `click`.       | string        | `hover`  |

**Comment by visvadw** [comment]
@benjycui Done reviewing, good translation. 👍

File: components/popover/index.en-US.md (hunk 22 above)
Comment (wenshiqi0) [inline components/popover/index.en-US.md]: Please review this one more.

File: components/popover/index.en-US.md [hunk 25]
```diff
@@ -5,19 +5,19 @@ type: Views
 english: Popover
 ---
 
-The bubble float card popped by clicking or hovering.   
+The floating card popped by clicking or hovering.
 
 ## When to use
 
-For more information or operations, put all into the popover card to show according to the behavior.
+A simple and pop-up menu to provide more information and operations of an action.
```
Comment (benjycui) [inline components/popover/index.en-US.md]: A simple popup menu to provide extra information or operations.

**Pull request merged**